packages/editor-ui/src/components/mixins/genericHelpers.ts

```diff
@@ -77,12 +77,12 @@ export const genericHelpers = mixins(showMessage).extend({
 		async callDebounced (...inputParameters: any[]): Promise<void> { // tslint:disable-line:no-any
 			const functionName = inputParameters.shift() as string;
 			const debounceTime = inputParameters.shift() as number;
-			const trailing = inputParameters.shift() as boolean;
+			const trailing = inputParameters.includes(true);
```

Comment:
can we find a better approach here? like passing a parameter object or checking parameter directly

Reply:
Agreed and updated. Now I passing parameter object to be more precise.